src/main/java/com/stevedutch/assignment14/domain/Message.java

```diff
@@ -17,17 +16,13 @@
 @Table(name = "messages")
 public class Message {
 
-//	@SerializedName("messageText") in Jackson it would be @JsonProperty("messageText")
 	private String messageText;
 
-//	@SerializedName("messageId")
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Column(name = "message_id")
-	private Long messageId;
```

Comment:
name message_id is redundant, since you are storing messages in the table, it could be just called id